compiler-rt/test/asan/TestCases/Windows/aligned_mallocs.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clang_cl_asan %Od %s %Fe%t
-// RUN: %run %t
+// RUN: not %run %t | FileCheck %s
```

Comment:
The ASan errors get printed to stderr. We need to redirect the output so FileCheck sees it. In other words, this line should be

```
// RUN: not %run %t 2>&1 | FileCheck %s
```